coordinator-external/src/main/scala/com/socrata/datacoordinator/external/DataCoordinatorError.scala

```diff
@@ -0,0 +1,110 @@
+package com.socrata.datacoordinator.external
+
+sealed class DataCoordinatorError
```

Comment:
Looking at the package declaration and the reference to some of these String Constants I am assuming these Errors are the ones surfaced to external clients?   If they are, is there a reason why the functionality to compose the message is not within the class itself?